src/main/java/dev/codesupport/web/api/data/repository/CountryRepository.java

```diff
@@ -0,0 +1,8 @@
+package dev.codesupport.web.api.data.repository;
+
+import dev.codesupport.web.api.data.entity.CountryEntity;
+import dev.codesupport.web.common.data.repository.CrudRepository;
+
+public interface CountryRepository extends CrudRepository<CountryEntity, Long> {
```

Comment:
Empty file?

Reply:
Yep, just how spring does Repos.  I don't need any additional queries beyond the default ones provided, I just need the interface so the default ones are created.

Reply:
Neat